### ISSUE TITLE
CSV paste in REPL functionality missing

### ISSUE THREAD

**Issue by huguesv** [Microsoft/PTVS#2688]
The test `ReplWindowTests.CsvPaste` checks that if the clipboard contains data in regular text and `DataFormats.CommaSeparatedValue` formats, that the CSV format is used when `Edit.Paste` command is issued.

The test fails because the text format is pasted.

I looked at the interactive window implementation:
https://github.com/dotnet/interactive-window/blob/3114957d329e725982d19dc04db6bfb6152f88b0/src/Microsoft.VisualStudio.InteractiveWindow/InteractiveWindow.UIThreadOnly.cs#L2660

There is no check for CSV format.

We probably should just delete this test.

**Comment by zooba** [comment]
We handle the CSV transformation ourselves - see `PythonCommonInteractiveEvaluator.FormatClipboard`

**Comment by huguesv** [comment]
The test is setting the clipboard data object to a memory stream, but the paste casts it to a string.

Apparently Excel used to use a stream for `DataFormats.CommaSeparatedvalue`, but I tried with the latest Excel and it both copies to clipboard and paste from clipboard as a string type.

So I'll change our test code to set it as a string instead of a stream.

**Comment by huguesv** [comment]
Turns out it's a WPF vs. Windows Forms Clipboard implementation difference when it comes to CSV format.

I've tested all permutations:

WPF reads the CSV format from clipboard as a string type always. It doesn't matter if clipboard was set using WPF or Windows Forms clipboard, in either string or stream format.  This is clearly the most consistent and best option for reading CSV from clipboard.

Windows Forms reads the CSV format from clipboard as a stream type but for one exception. If clipboard was set using Windows Forms clipboard in string format, then it reads it as string type, all other cases (WPF string or stream, Windows Forms stream) are returned as stream.

The problem here is that our code in `PythonCommonInteractiveEvaluator.FormatClipboard` is using the Windows Forms clipboard.

My guess is that the original code must have used the WPF clipboard and then was moved at some point, light bulb was used to add the using (or using was already there) and it got switched to the Windows Forms clipboard.
